homeassistant/components/tradfri/base_class.py

```diff
@@ -71,7 +79,7 @@ def _observe_update(self, device):
         self._refresh(device)
         self.async_write_ha_state()
 
-    def _refresh(self, device):
+    def _refresh(self, device: Command) -> None:
```

Comment:
There are other method on this file that could be typed, like `def _observe_update(self, device: Command) -> None:`

Reply:
yes but this PR is about config_flow.py there is another concentrating on base_class.py